blueprints/data-solutions/vertex-mlops/variables.tf

```diff
@@ -0,0 +1,144 @@
+  description = "BigQuery Dataset to store the training data."
+  type        = string
+  default     = null
+}
+
+
+variable "group_iam" {
+  description = "Authoritative IAM binding for the project, in {GROUP_EMAIL => [ROLES]} format."
+  type        = map(list(string))
+  default     = {}
+  nullable    = false
+}
+
+variable "identity_pool_claims" {
+  description = "Claims to be used by Workload Identity Federation (i.e.: attribute.repository/ORGANIZATION/REPO). If a not null value is provided, then google_iam_workload_identity_pool resource will be created."
+  type        = string
+  default     = null
+}
+
+variable "labels" {
+  description = "Labels to be assigned at project level."
+  type        = map(string)
+  default     = {}
+}
+
+variable "location" {
+  description = "Location used for multi-regional resources."
+  type        = string
+  default     = "eu"
+}
+
+variable "network_config" {
+  description = "Shared VPC network configurations to use. If null networks will be created in projects with preconfigured values."
+  type = object({
+    host_project      = string
+    network_self_link = string
+    subnet_self_link  = string
+  })
+  default = null
+}
+
+variable "notebooks" {
+  description = "Vertex AI workbenchs to be deployed."
+  type = map(object({
+    owner            = string
+    region           = string
+    subnet           = string
+    internal_ip_only = optional(bool, false)
+    idle_shutdown    = optional(bool)
+  }))
+  default = {}
+}
+
+variable "prefix" {
+  description = "Prefix used for the project id."
+  type        = string
+  default     = null
+}
+
+variable "project_create" {
+  description = "Provide values if project creation is needed, uses existing project if null. Parent is in 'folders/nnn' or 'organizations/nnn' format."
+  type = object({
+    billing_account_id = string
+    parent             = string
+  })
+  default = null
+}
+
+variable "project_id" {
+  description = "Project id, references existing project if `project_create` is null."
+  type        = string
+}
+
+variable "project_services" {
+  description = "List of core services enabled on all projects."
+  type        = list(string)
+  default = [
+    "aiplatform.googleapis.com",
+    "artifactregistry.googleapis.com",
+    "bigquery.googleapis.com",
+    "cloudbuild.googleapis.com",
+    "compute.googleapis.com",
+    "datacatalog.googleapis.com",
+    "dataflow.googleapis.com",
+    "iam.googleapis.com",
+    "monitoring.googleapis.com",
+    "notebooks.googleapis.com",
+    "secretmanager.googleapis.com",
+    "servicenetworking.googleapis.com",
+    "serviceusage.googleapis.com"
+  ]
+}
+
+variable "region" {
+  description = "Region used for regional resources."
+  type        = string
+  default     = "europe-west4"
+}
+
+variable "repo_name" {
+  description = "Cloud Source Repository name. null to avoid to create it."
+  type        = string
+  default     = null
+}
+
+variable "sa_mlops_name" {
```

Comment:
is valuable setting the name of the SA? I would remove it.

Reply:
This allows to use a different sufix (not prefix) per environment. Maybe this is something to iterate in the future